components/tree/style/mixin.less

```diff
@@ -4,7 +4,7 @@
 @select-tree-prefix-cls: ~'@{ant-prefix}-select-tree';
 @tree-motion: ~'@{ant-prefix}-motion-collapse';
 @tree-node-padding: (@padding-xs / 2);
-@tree-node-hightlight-color: inherit;
+@tree-node-highlight-color: inherit;
```

Comment:
It will be a breaking change, you can keep `@tree-node-hightlight-color` for compatibility.